src/index.js

```diff
@@ -111,13 +117,41 @@ $('#submit_team-member').on('click', function(event) {
   }
 })
 
+$('#submit_book-room-button').on('click', function() {
+  $('.div_available-rooms').remove();
+  selectedDate = formatSelectedDate($('#select-date').val());
+  let roomType = $('#select-room-type').val();
+  console.log(roomType);
+  if (selectedDate !== '' ) {
+    displayAvailableRooms(selectedDate, roomType);
+  } else {
+    $('#submit_book-room-button, .button_new-search, .error_no-rooms')
+      .toggleClass('hide');
+  }
+})
+
+$('.button_new-search').on('click', function() {
+  $('#select-date').val('');
+  $('#select-room-type').val('any');
+  $('#submit_book-room-button, .button_new-search, .error_no-rooms')
+    .toggleClass('hide');
+})
+
+$('.main_guest-page').on('click', '.div_available-rooms', function(event) {
+  event.preventDefault();
+  let postBody = user.makeBooking(selectedDate,
+    event.target.closest('.div_available-rooms').dataset.roomnumber);
+  postNewBooking(postBody);
+  $(event.target.closest('.div_available-rooms')).remove();
+})
 
 // HANDLERS
 
 function loadManagerPageDisplay() {
   $('#manager-dashboard-occupancy').text(`${tapeChart.getOccupancy(today)}%`);
   $('#manager-dashboard-revenue').text(`$${tapeChart.getDaysRevenue(today)}`);
-  $('#manager-dashboard-availability').text(`${tapeChart.getAvailableRoomsByDate(today).length}`);
+  $('#manager-dashboard-availability').text
+    (`${tapeChart.getAvailableRooms(today).length}`);
 }
 
 function loadGuestPageDisplay() {
```

Comment:
I like this way this dynamically finds users, if there were more added it will still work. I might consider reworking my code using this.